http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/DataprocRuntimeMonitor.scala

```diff
@@ -72,11 +72,17 @@ class DataprocRuntimeMonitor[F[_]: Parallel](
         LeoLenses.cloudContextToGoogleProject.get(runtimeAndRuntimeConfig.runtime.cloudContext),
         new RuntimeException("this should never happen. Dataproc runtime's cloud context should be a google project")
       )
-      cluster <- googleDataprocService.getCluster(
-        googleProject,
-        dataprocConfig.region,
-        DataprocClusterName(runtimeAndRuntimeConfig.runtime.runtimeName.asString)
-      )
+      cluster <- googleDataprocService
+        .getCluster(
+          googleProject,
+          dataprocConfig.region,
+          DataprocClusterName(runtimeAndRuntimeConfig.runtime.runtimeName.asString)
+        )
+        .recoverWith { case _: com.google.api.gax.rpc.PermissionDeniedException =>
```

Comment:
we're recovering here so that Leo will eventually move the runtime to `Error` instead of having it stuck in a specific status